blockchains/smartchain/assets/0xdbf8265b1d5244a13424f13977723acf5395eab2/info.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "Wagerr",
+  "website": "https://www.wagerr.com",
+  "description": "BEP20 tokenized version of wagerr",
+  "explorer": "https://bscscan.com/token/0xdbf8265b1d5244a13424f13977723acf5395eab2",
+  "type": "BEP20",
+  "symbol": "WGR",
+  "decimals": 18,
+  "status": "active",
+  "id": "0xdbf8265b1d5244a13424f13977723acf5395eab2"
```

Comment:
Change this to `"id": "0xdBf8265B1d5244A13424f13977723AcF5395eAB2"`

Reply:
done